core/src/main/java/org/kiwix/kiwixmobile/core/main/FileReader.kt

```diff
@@ -15,31 +15,20 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.core.main;
+package org.kiwix.kiwixmobile.core.main
 
-import android.content.Context;
-import java.io.BufferedReader;
-import java.io.InputStream;
-import java.io.InputStreamReader;
+import android.content.Context
+import java.io.BufferedReader
+import java.io.IOException
 
-public class FileReader {
-
-  public String readFile(String filePath, Context context) {
-    try {
-      StringBuilder buf = new StringBuilder();
-      InputStream json = context.getAssets().open(filePath);
-      BufferedReader in =
-        new BufferedReader(new InputStreamReader(json, "UTF-8"));
-      String str;
-
-      while ((str = in.readLine()) != null) {
-        buf.append(str);
-      }
-
-      in.close();
-      return buf.toString();
-    } catch (Exception e) {
-      return "";
+class FileReader {
+  fun readFile(filePath: String?, context: Context): String {
+    return try {
+      val json = context.assets.open(filePath)
+      return json.bufferedReader().use(BufferedReader::readText)
```

Comment:
this return is a bit weird when we are already returning the try catch.

I would inline `json` and put `buffered` and `use` on their own lines 

and then to match up catch so it is 1 line like the try section something like
```
"".also{ e.printStackTrace() }
```

Reply:
<img width="592" alt="Screenshot 2020-08-20 at 4 56 56 PM" src="https://user-images.githubusercontent.com/43576162/90764603-29463f80-e306-11ea-9272-ede6abb246e4.png">
@macgills is this the right way?

Reply:
the `also` can all be 1 line, there should be no variables in the try but all the statements are on my preffered lines
they
should
be
[chainedCalls](https://kotlinlang.org/docs/reference/coding-conventions.html#chained-call-wrapping)

Reply:
<img width="592" alt="Screenshot 2020-08-20 at 5 12 44 PM" src="https://user-images.githubusercontent.com/43576162/90765913-5e539180-e308-11ea-8c5d-6b48c566c65b.png">
@macgills Is this ok?

Reply:
perfection, the only thing we could do is turn this into an expression body, press `alt+return` on `return` and it'll be an option